align-star.wdl

```diff
@@ -0,0 +1,30 @@
+import "tasks/star.wdl" as star_task
+import "tasks/samtools.wdl" as samtools
+
+workflow Mapping {
```

Comment:
The names are now conflicting, Maybe Call the `AlignStar`? The other one should be called `AlignBwaMem` ofcourse ;)